odl/deform/linearized_deform.py

```diff
@@ -0,0 +1,538 @@
+# Copyright 2014-2016 The ODL development group
```

Comment:
The file name should be just `linearized.py` since the import path will contain the `deform` part: `odl.deform.linearized`. That's as explicit as it can get.